web/app/scripts/navbar/navbar-partial.html

```diff
@@ -23,7 +23,7 @@
                         {{ ctl.recordTypeSelected.plural_label }}
                         <span class="caret"></span></a>
                     <ul class="dropdown-menu">
-                        <li class="nav-rt-item" ng-repeat="recordType in ctl.recordTypeResults | orderBy:'label'">
+                        <li class="nav-rt-item" ng-repeat="recordType in ctl.recordTypeResults | orderBy:'plural_label'">
```

Comment:
I think it might make sense to remove the angular list reorder filter, and just handle results ordering django-side, to speed up the client a bit. If needed, we could send in an ordering parameter instead of relying on a default sort order from the API.